users/config.rst

```diff
@@ -193,6 +193,10 @@ Folder Element
         <markerName>.stfolder</markerName>
         <copyOwnershipFromParent>false</copyOwnershipFromParent>
         <modTimeWindowS>0</modTimeWindowS>
+        <maxConcurrentWrites>2</maxConcurrentWrites>
+        <disableFsync>false</disableFsync>
+        <blockPullOrder>standard</blockPullOrder>
+        <copyRangeMethod>standard</copyRangeMethod>
```

Comment:
The same 4 lines should probably be added to the folder element under https://github.com/syncthing/docs/blob/new-docs/users/config.rst#config-file-format (i.e. between the lines 71 and 72).